cmd/kops/replace.go

```diff
@@ -138,6 +138,12 @@ func RunReplace(ctx context.Context, f *util.Factory, out io.Writer, c *ReplaceO
 						if !c.Force {
 							return fmt.Errorf("cluster %v does not exist (try adding --force flag)", clusterName)
 						}
+
+						err = cloudup.PerformAssignments(v, cloud)
+						if err != nil {
+							return fmt.Errorf("error populating configuration: %v", err)
```

Comment:
This code generally lacks error wrapping, but suggest we do it for any new lines anyway.

```suggestion
							return fmt.Errorf("error populating configuration: %w", err)
```